EIPS/eip-draft_my_own_messages.md

```diff
@@ -0,0 +1,145 @@
+- Finally, you can ask and get tips for your words directly into your wallet.
+
+I know, My Own Messages (MOM) sounds like _mom_. And yes, pun intended :)
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+My Own Messages (MOM) is a standard to notarize and share messages using Ethereum. You can give voice to your smart contracts, send messages to the world, create a certified blog with your ideas, and so on.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+As a _developer_ or _pool's owner_, I'd like to send messages to my users in a decentralized way. They must be able to easily verify my role in the smart contract context (owner, user, and so on) and they must be able to do it without relying on external, insecure and hackable social media sites (Facebook, Twitter, you name it). Also, I'd like to read messages from my userbase, in the same secure and verifiable manner.
+
+As a _user_, I want a method to easily share my thoughts and idea, publish content, send messages, receive feedbaks, receive tips, and so on, wihout dealing with any complexity: just write a message, send it and it's done. Also, I want to write to some smart contract's ower or to the sender of some transaction.
+
+As an _explorer service_, I want to give my users an effective way to read informations by smart contract owners and a place to share ideas and informations, without using third parties services (i.e. Etherscan uses Disqus, and so on)
+
+And in _any role_, I want a method that does not allow scams - transactions without values, no smart contract's address to remember or to fake - and it does not allow spam - it's cheap but not free, and even if you can link/refer other accounts, you cannot send them messages directly, and others must explicitly follow and listen to your transactions if they want to read your messages.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+## Valid MOM transaction's structure
+
+| ATTRIBUTE | VALUE |
+|:--------|:------------|
+| `from` | `MUST` be the tx signer |
+| `to` | `MUST` be the tx signer |
+| `value` |  `MUST` be 0 wei |
+| `data` | `MUST` be at least 1 byte. First byte is the operational code, then it comes the content. |
```

Comment:
Recommend using RFC 2119 style here (and consider adding the RFC 2119 blurb at the top of the Specification section).